docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -375,9 +376,60 @@ N/A
 
 **Usage notes:**
 
-* ``jointags()`` pivots using the set of leaf tags only. For example if nodes in the working set have the tag ``#foo.bar.baz``, ``jointags()`` will return other nodes with ``#foo.bar.baz``, but not nodes with ``#foo.bar`` or ``#foo`` alone.
-* ``jointags()``, like ``refs()``, can be useful to "explore" other nodes that share some analytical assessment (tag) with the working set of nodes, but may return a large number of nodes. It may be more efficient to narrow the scope of the query using ``totags()`` in combination with a filter operator (e.g., to limit the specific tags selected) followed by ``fromtags()``.
-* The ``limit=`` parameter can be provided as input to the ``jointags()`` operator itself when using Operator syntax. Alternately the ``limit()`` operator_ can be used after the ``jointags()`` operator (in either Operator or Macro syntax) to specify a limit on the number of nodes returned.
+* ``jointags()`` joins using the set of leaf tags only. For example if nodes in the working set have the tag ``#foo.bar.baz``, ``jointags()`` will return other nodes with ``#foo.bar.baz``, but not nodes with ``#foo.bar`` or ``#foo`` alone.
+* ``jointags()``, like ``refs()``, can be useful to "explore" other nodes that share some analytical assessment (tag) with the working set of nodes, but may return a large number of nodes.
+It may be more efficient to narrow the scope of the query using ``totags()`` in combination with a filter operator (e.g., to limit the specific tags selected) followed by ``fromtags()``.
+* The ``limit=`` parameter can be provided as input to the ``jointags()`` operator itself when using Operator syntax.
+Alternately the ``limit()`` operator_ can be used after the ``jointags()`` operator (in either Operator or Macro syntax) to specify a limit on the number of nodes returned.
```

Comment:
Should we clarify how "limit" works for jointags() vs pivottags()? e.g., for a join-type operation, "limit" represents the maximum number of nodes **added** to the results (e.g., the original working set, plus "limit" number of additional nodes)? Similar for pivottags() below (see).